src/Manager/SMSManager.php

```diff
@@ -33,6 +33,20 @@ public function __construct(
     ) {
     }
 
+    /** @throws \Exception */
+    public function sendAffiliationCodeSms(Beneficiaire $beneficiary): void
+    {
+        $telephone = $beneficiary->getUser()?->getTelephone();
+        $code = $this->getRandomSmallSmsCode();
+        $beneficiary->setRelayInvitationSmsCode($code);
```

Comment:
Maybee a method could encapsulate the two, maybee the setRelayIndivationSmsCOde should update the sentAt Date